internal/driver/gomobile/repository.go

```diff
@@ -0,0 +1,36 @@
+// +build ios android
+
+package gomobile
+
+import (
+	"fyne.io/fyne"
+)
+
+type mobileFileRepo struct {
```

Comment:
consider adding type assertions to ensure this type still implements all the expected interfaces - I think it no longer implements `ListableRepository`

Reply:
a4aa5511302da040b88069e18126780b65ef4748